content/03kapitel.tex

```diff
@@ -2,6 +2,51 @@
 
 \chapter{Roadmap}
 
-\section{Epic 1}
+\section{Überblick}
+Nachdem die Requirements, Rollenaufteilung und Organisation des Projektes geklärt sind, geht es darum zu klären, was später in den tatsächlichen Sprints geschehen soll.
+Als Orientierungshilfe werden hierbei die Requirements des Projektes geordnet und in Product Increments aufgeteilt.
 
-\section{Epic 2}
+In diesem Kontext können \ac{PI}s als die geplanten größeren Versionssprünge unserer Plattform verstanden werden.
+Wichtig ist es hierbei hervorzuheben, dass \ac{PI}s nicht die einzelnen Sprintziele darstellen.
+Diese im Voraus (mit festen zeitlichen Markern) festzulegen hätte wenig mit der im Team erzielten agilen Projektorganisation zu tun.
+Vielmehr stellen die \ac{PI}s des Projektes den geplanten Pfad dar, in welchem die Requirements abgearbeitet werden sollen.
+
+\section{Product Increments}
+Aus der obig erläuterten Überlegung leitet sich sie in Abbildung \ref{fig:roadmap} dargestellte Roadmap ab.
+Die verschiedenen \ac{PI}s sollen nun in den folgenden Abschnitten erläutert werden.
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.3\linewidth]{roadmap.png}
+    \caption{Roadmap des Projektes}
+    \label{fig:roadmap}
+\end{figure}
+
+\subsection{Initial Infrastructure}
+Ziel des \ac{PI}s ``Initial Infrastructure''\footnote{dt. Anfängliche Infrastruktur} ist es die Grundstruktur für die Plattform aufzusetzen und Workflow Elemente wie eine \ac{CI/CD} Pipeline, das Kubernetes Cluster und die initiale Servicestruktur zu erstellen.
+Fragen rund um, wie die Service-Architektur und Infrastruktur aufzusetzen ist, gilt hierbei zu klären.
+Am Ende dieses \ac{PI}s soll das Ergebnis eine funktionsfähige Infrastruktur sein, auf welcher in kommenden Sprints entwickelt und Sprint-Ergebnisse ausgeliefert werden können.
+
+\subsection{Article Views and Base Homepage}
+Ziel des \ac{PI}s ``Article Views and Base Homepage''\footnote{dt. Homepage und Einsichtseiten} ist es die wichtigsten Seiten der Applikation zu erstellen.
```

Comment:
```suggestion
Ziel des \acp{PI} \enqoute{Article Views and Base Homepage}\footnote{dt. Artikel-Seiten und Startseite} ist es die wichtigsten Seiten der Applikation zu erstellen.
```
* [ ] Replace all Quotes with \enqoute{}

Reply:
fixed, 
![image](https://user-images.githubusercontent.com/54294591/167296856-92ae4cc7-affe-4037-ba94-313c2a2962b9.png)
still looks like we're using english quotes though

Reply:
Never mind, forgot to merge your changes